### DISCUSSION TITLE
Missing methods in Options

### DISCUSSION THREAD

**Discussion by Lurler** [sebastienros/jint#1483] [Q&A]
Hey!

I'm not sure if I'm doing something wrong, but it's seems a lot of methods specified in the documentation are missing from the ``Options`` class in the actual library.

This includes:
```csharp
options.LimitMemory(4_000_000);
options.EnableModules(@"C:\Scripts");
```
etc.

Navigating (F12) to the ``Options`` class confirms it. Is there a difference to what version I should use or am I missing something? I simply included the official nuget package in my project. Everything works just fine, except those missing methods in the options.

**Comment by lahma** [comment]
Is the "official package" [3.0.0-beta-2047](https://www.nuget.org/packages/Jint/3.0.0-beta-2047) which should have these?

**Reply by Lurler** [reply]
Ah, I see what the issue is. Thank you for pointing it out!

The default package on NuGet is centuries old (version 2.x from 2017)... I guess it would be a good idea to update the default package there to a newer version.

**Reply by lahma** [reply]
Default package is the latest non-beta release, so one just has to read the repo readme which has this information.

**Reply by Lurler** [reply]
I might have missed that part, but anyway, thank you for the help :)

Great library so far!
